packages/appstore/src/components/containers/listing-container.tsx

```diff
@@ -14,9 +14,7 @@ type TListingContainerProps = {
 type TOptionsProps = Pick<TListingContainerProps, 'title' | 'description' | 'is_deriv_platform'>;
 type TSwitcherProps = Pick<TListingContainerProps, 'is_deriv_platform'>;
 
-const Options = ({ title, description, is_deriv_platform }: TOptionsProps) => {
-    if (!is_deriv_platform) return null;
-
+const Options = ({ title, description }: TOptionsProps) => {
```

Comment:
@mitra-deriv can you check line 42. we are still passing is_deriv_platform as a prop

Reply:
@amina-deriv that one is needed for showing account switcher 🙏